skietbaan_fe/src/components/ScoreCapture.js

```diff
@@ -279,94 +295,146 @@ export default class search extends Component {
+        <div className={this.state.showCamera || this.state.ImageTaken ? "page-content-video" : "page-content"}>
+          <div className={this.state.showCamera || this.state.ImageTaken ? "hidden" : ""}>
+
+            <div className="label-score">
+              <div className="centre-label">
+                <label className="scorelabel">Type in score</label>
+              </div>
+              <div className={this.state.validScore ? "hidden" : "invalidScore"}>.</div>
               <div className="input-container">
                 <input type="number" id="scoreInput" min="0" step="1" name="score" className="score"
-                  onChange={this.handleScore} placeholder="Score"></input>
+                  onChange={this.handleScore}></input>
               </div>
-              <div className={this.state.validScore ? "hidden" : "invalid"}>Please enter valid score </div>
+
             </div>
+            <div className={this.state.scoreEntered ? "":"hidden"}>
             <div className="centre-label">
               <label className="label-competition">Select Competition</label>
-               <div className={this.state.validCompetition ? "hidden" : "invalid"}>Please select a competition</div>
             </div>
+
+            <div className={this.state.validCompetition ? "hidden" : "invalidComp"}>.</div>
             <div className="competition-container">
               {competitionItem}
             </div>
+            </div>
             <div className={this.state.scoreSaved ? "sucess-container" : "hidden"}>
               <div className="success"> Score Saved successfully </div>
             </div>
-            <div className="submit-container">
-              <div className={this.state.ImageTaken ? "hidden" : "submit-button-elements"}>
-                <div className="button-hover">
+          </div>
+          <div className="submit-container">
+            <div className={this.state.ImageTaken || this.state.showCamera 
+              || !this.state.scoreEntered ? "hidden" : "submit-button-elements"}>
+              <div className="button-hover">
 
-                  <img src={require('../components/assets/scoreCapture.png')} 
-                    id="btnScoreCapture" onClick={() => this.CameraClicked()} alt=''></img>
+                <img src={require('../components/assets/scoreCapture.png')}
+                  id="btnScoreCapture" className="btnScoreCapture"
+                  onClick={() => this.CameraClicked()} alt=''></img>
+              </div>
+              <label className="labelIcon">Capture score</label>
+            </div>
+            <div className={(this.state.showCamera && !this.state.ImageTaken) 
+              || this.state.ImageTaken || !this.state.scoreEntered ? "hidden" : "submit-button-elements"}>
+              <div className="button-hover ">
+                <img src={require('../components/assets/submitScore.png')} onClick={() => this.GetLocation()}
```

Comment:
You have used this path `../components/assets/submitScore.png` 3 times or more. 
please make use of a constant

Reply:
You don't have to right but let us say the image changes to a different one, you have to find each occurrence or you could change it once in the constants

Reply:
The image is never being changed to a different one its just being hidden. I tried making the image path a constant but its not working